src/nav.json

```diff
@@ -241,6 +241,16 @@
             }
           ]
         },
+        {
+          "name": "References",
```

Comment:
The Glossary page needs a different home. It doesn't make sense to add a new section for this page. Also, this section name is confusing, since we already have a top-level "Reference" collection. For now, it might be best to add it under "About Koji for Developers". We will do a more comprehensive IA review soon, though.

Reply:
Moved to "About Koji for Developers".